entry_points.ini

```diff
@@ -0,0 +1,37 @@
+#
+# MIT License
+#
+# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the 'Software'),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+# ADD NEW SCRIPTS HERE
+# Example:
+#
+#   Call bar() inside of csm.foo on the CLI using a command called 'test' by adding:
+#
+#   test = csm.foo:bar
+#
+#   Call baz() inside of csm.bax on the CLI using a command called tester, which requires two
+#   optional dependencies listed in [project.optional-dependencies] in pyproject.toml.
+#
+#   test = csm.foo:bar [bob, alice]
+[console_scripts]
+canu = canu.cli:cli [network_modeling]
```

Comment:
also need 
```
canu-inventory = canu.inventory.ansible:ansible_inventory
```
to comply with recent changes.. (this might trigger updates elsewhere)